anime_downloader/sites/helpers/selescrape.py

```diff
@@ -0,0 +1,193 @@
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from anime_downloader.const import get_random_header
+from selenium.webdriver.common.by import By
+from urllib.parse import urlencode
+from urllib.parse import urlsplit
+from selenium import webdriver
+from bs4 import BeautifulSoup
+from logging import exception
+from sys import platform
+import requests
+import os
+import click
+import time
+import json
+
+
+def get_data_dir():
+    '''
+    Gets the folder directory selescrape will store data, 
+    such as cookies or browser extensions and logs.
+    '''
+    APP_NAME = 'anime downloader'
+    return os.path.join(click.get_app_dir(APP_NAME), 'data')
+
+
+def open_config():
+    from anime_downloader.config import Config
+    return Config
+
+    
+data = open_config()
+
+
+def get_browser_config():
+    '''
+    Decides what browser selescrape will use.
+    '''
+    os_browser = { #maps os to a browser
+    'linux':'firefox',
+    'darwin':'chrome',
+    'win32':'chrome'
+    }
+    for a in os_browser:
+        if platform.startswith(a):
+            browser =  os_browser[a]
+        else:
+            browser = 'chrome'
+    value = data['dl']['selescrape_browser']
+    value = value.lower() if value else value
+    if value in ['chrome', 'firefox']:
+        browser = value
+    return browser
+
+
+def get_browser_executable():
+    value = data['dl']['selescrape_browser_executable_path']
+    executable_value = value.lower() if value else value
+    return executable_value
+
+
+def get_driver_binary():
```

Comment:
This and `get_browser_executable` are basically the same code. Merge these functions